galaxy/templates/cronjob-maintenance.yaml

```diff
@@ -21,12 +21,18 @@ spec:
             image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
             imagePullPolicy: {{ .Values.image.pullPolicy }}
             # delete all tmp files older than walltime limit
-            args:
+            command:
```

Comment:
I suppose the error arose because of how are YAMLs parsed. Some parts of the command included special symbols e.g. `: ; \ "` and in the end, whole command was not syntactically correct. However, now it's okay

Reply:
Makes sense.